.github/workflows/update-latest-images.yml

```diff
@@ -0,0 +1,74 @@
+name: Update base images
+
+
+on:
+  workflow_dispatch:
+    inputs:
+      params:
+        description: 'Additional parameters to updateBaseImages.sh'
+        default: ''
+        required: false
+      branch:
+        description: 'Project branch'
+        default: 'master'
+        required: true
+
+jobs:
+  build:
+    name: Check if images need to be updated to latest version
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: "Checkout source code"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: "Set up QEMU"
```

Comment:
do we need qemu and buildx for a bash script that doesn't build any multi-arch containers?

Reply:
I guess not